cpp/src/io/comp/unsnap.cu

```diff
@@ -705,11 +702,6 @@ CUDF_KERNEL void __launch_bounds__(block_size)
     results[strm_id].bytes_written = s->uncompressed_size - s->bytes_left;
     results[strm_id].status =
       (s->error == 0) ? compression_status::SUCCESS : compression_status::FAILURE;
-    if (log_cyclecount) {
-      results[strm_id].reserved = clock() - s->tstart;
-    } else {
-      results[strm_id].reserved = 0;
-    }
```

Comment:
Has the cycle count logic been relocated elsewhere, or is it simply being removed?

Reply:
Good question, I forgot to comment on this.
It's removed: It's unused and keeping it just adds noise to the future API changes.